src/index.ts

```diff
@@ -1,85 +1,85 @@
 import { provideCore, AnswersConfig, AdditionalHttpHeaders } from '@yext/answers-core';
 import HttpManager from './http-manager';
 import ReduxStateManager from './redux-state-manager';
-import AnswersHeadless from './answers-headless';
+import SearchHeadless from './search-headless';
 import { createBaseStore } from './store';
 import HeadlessReducerManager from './headless-reducer-manager';
 import { DEFAULT_HEADLESS_ID } from './constants';
 import { SessionTrackingState } from './models/slices/sessiontracking';
-import * as answersUtilities from './answers-utilities';
+import * as searchUtilities from './search-utilities';
 import { getHttpHeaders } from './utils/client-sdk-utils';
 
 export * from './answers-core-re-exports';
 export * from './models';
 export * from './constants';
 export * from './utils/filter-creators';
 export * from './utils/types';
-export { answersUtilities };
+export { searchUtilities };
 
 /**
- * The configuration for an AnswersHeadless instance.
+ * The configuration for an SearchHeadless instance.
  *
  * @public
  */
 export type HeadlessConfig = AnswersConfig & {
   /**
-   * The ID of the AnswersHeadless instance.
+   * The ID of the SearchHeadless instance.
    *
    * @remarks
    * Must be different from {@link DEFAULT_HEADLESS_ID}.
    */
   headlessId?: string,
   /**
    * The verticalKey associated with the vertical to manage. If none is provided,
-   * Answers Headless will manage universal search.
+   * Search Headless will manage universal search.
    */
   verticalKey?: string
 };
 
-let firstHeadlessInstance: AnswersHeadless;
+let firstHeadlessInstance: SearchHeadless;
 const store = createBaseStore();
 const headlessReducerManager = new HeadlessReducerManager();
 
 /**
- * Supplies a new instance of {@link AnswersHeadless}, using the provided configuration.
+ * Supplies a new instance of {@link SearchHeadless}, using the provided configuration.
  *
  * @param config - The apiKey, experienceKey, etc. needed to set up a front-end Answers
  *                 experience.
- * @returns The newly created instance of {@link AnswersHeadless}
+ * @returns The newly created instance of {@link SearchHeadless}
  *
  * @public
  */
-export function provideAnswersHeadless(config: HeadlessConfig): AnswersHeadless;
+export function provideSearchHeadless(config: HeadlessConfig): SearchHeadless;
```

Comment:
maybe these can be renamed to `provideHeadless`, like we have `provideCore`? that would future-proof us from any other re-brandings